pkg/events/events.go

```diff
@@ -205,6 +231,12 @@ const (
 	CapturePcap
 )
 
+// Rules events
+const (
+	StartRulesID ID = 6000
```

Comment:
hmm, I thought that considering we want a specific boundary for the id of the rule, enforcing a range, it would be best to have constants here, and perhaps validate them. But yeah, this code seems lost here.

Reply:
I don't know if it's reasonable to reach 1000 rules, and maybe it's even wise to set this hard limit, but if we do it should at least be documented.
I think we can keep the consts here as long as we validate for them.

Reply:
+1 on keeping it here, together with other limits.